### PR TITLE
[SSL] Geo Key Manager docs section

### PR DESCRIPTION
**Pull request by kodster28**
PCX-4550

### CODE REVIEW

File: content/ssl/_partials/_geokey-manager-v1.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+_build:
+  publishResources: false
+  render: never
+  list: never
+---
+
+The first version of Geo Key Manager supports 3 regions: U.S., EU, and a set of High Security Data Centers. If you would like to restrict your private key to another country or region, [apply for the closed beta](https://www.cloudflare.com/lp/geo-key-manager/) of the new version.
```
Comment (pedrosousa) [inline content/ssl/_partials/_geokey-manager-v1.md]:
```suggestion
The first version of Geo Key Manager supports 3 regions: U.S., E.U., and a set of High Security Data Centers. If you would like to restrict your private key to another country or region, [apply for the closed beta](https://www.cloudflare.com/lp/geo-key-manager/) of the new version. 
```

File: content/ssl/edge-certificates/geokey-manager/setup.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+{{<Aside type="note">}}
+
+Geo Key Manager v2 is only available through the Cloudflare API.
+
+{{</Aside>}}
+
+Geo Key Manager v2 gives customers flexibility when choosing the geographical boundaries of where their keys are stored.
+
+Using the `policy` field, customers can define policies which comprise of allow and block lists of countries or regions where the private key should be stored.
```
Comment (pedrosousa) [inline content/ssl/edge-certificates/geokey-manager/setup.md]:
```suggestion
Using the `policy` field, customers can define policies containing allow and block lists of countries or regions where the private key should be stored. 
```

File: content/ssl/edge-certificates/geokey-manager/setup.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+{{</Aside>}}
+
+Geo Key Manager v2 gives customers flexibility when choosing the geographical boundaries of where their keys are stored.
+
+Using the `policy` field, customers can define policies which comprise of allow and block lists of countries or regions where the private key should be stored. 
+
+To use Geo Key Manager v2 with the API, generally, follow the steps to [upload a custom certificate](/ssl/edge-certificates/custom-certificates/uploading/#using-the-api).
+
+When sending the [`POST`](https://developers.cloudflare.com/api/operations/custom-ssl-for-a-zone-create-ssl-configuration) request, include the `policy` parameter to define policies which comprise of allow and block lists of countries or regions where the private key should be stored.
```
Comment (pedrosousa) [inline content/ssl/edge-certificates/geokey-manager/setup.md]:
```suggestion
When sending the [`POST`](https://developers.cloudflare.com/api/operations/custom-ssl-for-a-zone-create-ssl-configuration) request, include the `policy` parameter to define policies containing allow and block lists of countries or regions where the private key should be stored.
```

File: content/ssl/edge-certificates/geokey-manager/setup.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+You also have access to the `geo_restrictions` parameter, which is mutually exclusive to the `policy` parameter and includes the following options:
```
Comment (pedrosousa) [inline content/ssl/edge-certificates/geokey-manager/setup.md]:
```suggestion
You also have access to the `geo_restrictions` parameter, which is mutually exclusive with the `policy` parameter and includes the following options:
```
Usually there's no preposition after "mutually exclusive", but I'd use "with".

File: content/ssl/edge-certificates/geokey-manager/setup.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+- `highest_security`([more details](/ssl/edge-certificates/geokey-manager/supported-options/#highest-security-data-centers))
+
+### Examples
+
+```json
+---
+header: Store private keys in EU and US
+---
+curl -X POST "https://api.cloudflare.com/client/v4/zones/<ZONE_ID>/custom_certificates" \
+     -H "X-Auth-Email: user@example.com" \
```
Comment (pedrosousa) [inline content/ssl/edge-certificates/geokey-manager/setup.md]:
```suggestion
     -H "X-Auth-Email: <EMAIL>" \
```

File: content/ssl/edge-certificates/geokey-manager/setup.md [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+        "type": "sni_custom"
+     }'
+```
+
+```json
+---
+header: Store private keys in the EU, but not in France
+---
+curl -X POST "https://api.cloudflare.com/client/v4/zones/<ZONE_ID>/custom_certificates" \
+     -H "X-Auth-Email: user@example.com" \
```
Comment (pedrosousa) [inline content/ssl/edge-certificates/geokey-manager/setup.md]:
```suggestion
     -H "X-Auth-Email: <EMAIL>" \
```

File: content/ssl/edge-certificates/geokey-manager/setup.md [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+        "certificate":"certificate",
+        "private_key":"<PRIVATE_KEY>",
+        "policy":"(country: US) and (region: EU)", 
+        "type": "sni_custom"
+     }'
+```
+
+```json
+---
+header: Store private keys in the EU, but not in France
```
Comment (pedrosousa) [inline content/ssl/edge-certificates/geokey-manager/setup.md]:
```suggestion
header: Store private keys in the E.U., but not in France
```

File: content/ssl/edge-certificates/geokey-manager/setup.md [hunk 8]
```diff
@@ -0,0 +1,106 @@
...
+
+- `us`
+- `eu`
+- `highest_security`([more details](/ssl/edge-certificates/geokey-manager/supported-options/#highest-security-data-centers))
+
+### Examples
+
+```json
+---
+header: Store private keys in EU and US
```
Comment (pedrosousa) [inline content/ssl/edge-certificates/geokey-manager/setup.md]:
```suggestion
header: Store private keys in the E.U. and the U.S.
```

File: content/ssl/edge-certificates/geokey-manager/supported-options.md [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+| WEUR | Western Europe |
+| SAM | South America |
+| ENAM | Eastern North America |
+| WNAM | Western North America |
+
+---
+
+## Available countries
+
+For customers with Geo Key Manager v2, you can use the `policy` parameter to individual countries as well. Cloudflare is constantly expanding the number of supported countries. To indicate a country, specify the two-letter (ISO 3166) country code.
```
Comment (pedrosousa) [inline content/ssl/edge-certificates/geokey-manager/supported-options.md]:
```suggestion
For customers with Geo Key Manager v2, you can use the `policy` parameter to specify individual countries as well. Cloudflare is constantly expanding the number of supported countries. To indicate a country, specify the two-letter (ISO 3166) country code.
```

File: content/ssl/edge-certificates/geokey-manager/supported-options.md [hunk 10]
```diff
@@ -0,0 +1,63 @@
...
+
+---
+
+## Highest security data centers
+
+For customers with both Geo Key Manager v1 and v2, you can use the `geo_restrictions` parameter to only choose Cloudflare's highest security data centers.
+
+The following aspects are unique to our highest security data centers, but the baseline security requirements for all data centers are also detailed in [our blog](https://blog.cloudflare.com/introducing-cloudflare-geo-key-manager/).
+
+### Pre-Scheduled, Biometric Controlled Facility Access
```
Comment (pedrosousa) [inline content/ssl/edge-certificates/geokey-manager/supported-options.md]: Not sure if we must keep the capitalization here or if these headings can use sentence case.

File: content/ssl/edge-certificates/geokey-manager/supported-options.md [hunk 11]
```diff
@@ -0,0 +1,63 @@
...
+
+Employees of Cloudflare permitted to access the facility must have previously scheduled a visit before access will be granted.
+
+Access to the entrance of the facility is controlled through the use of a biometric hand reader combined with an assigned access code.
+
+### Private Cages with Biometric Readers
+
+All equipment is in private cages with physical access controlled via biometrics and recorded in audit logs.
+Entrants have to pass through 5 separate readers before the cage can be accessed.
```
Comment (pedrosousa) [inline content/ssl/edge-certificates/geokey-manager/supported-options.md]:
```suggestion
Entrants have to pass through five separate readers before they can access the cage.
```
Hopefully, we can adjust this text...

File: content/ssl/edge-certificates/geokey-manager/supported-options.md [hunk 12]
```diff
@@ -0,0 +1,63 @@
...
+
+### Exterior Security Controls and Monitoring
+
+All points of ingress/egress are monitored by an intrusion detection system (IDS), with authorized users and access events archived for historical review.
+
+### Interior Security Controls and Monitoring
+
+Interior points of ingress/egress are controlled by the access control subsystem, with entry routed through a mantrap. All areas are monitored and recorded with closed-circuit television, with data kept for a minimum of thirty days.
+
+Exterior walls are airtight and may incorporate additional security measure such as reinforced concrete, Kevlar bullet board, vapor barriers, or bullet-proof front doors.
```
Comment (pedrosousa) [inline content/ssl/edge-certificates/geokey-manager/supported-options.md]:
```suggestion
Exterior walls are airtight and may incorporate additional security measures such as reinforced concrete, Kevlar bullet board, vapor barriers, or bullet-proof front doors.
```

File: content/ssl/edge-certificates/geokey-manager/setup.md [hunk 13]
```diff
@@ -0,0 +1,106 @@
+---
+pcx_content_type: how-to
+title: Setup
+weight: 1
+layout: single
+meta:
+    title: Setup - Geo Key Manager
+---
```
Comment (pedrosousa) [inline content/ssl/edge-certificates/geokey-manager/setup.md:9]: We should probably have a description here also.

File: content/ssl/edge-certificates/geokey-manager/supported-options.md [hunk 14]
```diff
@@ -0,0 +1,64 @@
+---
+pcx_content_type: reference
+title: Supported options
+weight: 2
+layout: single
+meta:
+    title: Supported options - Geo Key Manager
+---
```
Comment (pedrosousa) [inline content/ssl/edge-certificates/geokey-manager/supported-options.md:9]: We should probably have a description here also.

**Review by pedrosousa — APPROVED** [review]

File: content/ssl/edge-certificates/geokey-manager/setup.md [hunk 15]
```diff
@@ -0,0 +1,107 @@
...
+
+Using the `policy` field, customers can define policies containing allow and block lists of countries or regions where the private key should be stored. 
+
+To use Geo Key Manager v2 with the API, generally, follow the steps to [upload a custom certificate](/ssl/edge-certificates/custom-certificates/uploading/#using-the-api).
+
+When sending the [`POST`](https://developers.cloudflare.com/api/operations/custom-ssl-for-a-zone-create-ssl-configuration) request, include the `policy` parameter to define policies containing allow and block lists of countries or regions where the private key should be stored.
+
+{{<Aside type="note">}}
+
+You also have access to the `geo_restrictions` parameter, which is mutually exclusive to the `policy` parameter and is part of [Geo Key Manager v1](#geo-key-manager-v1).
```
Comment (kodster28) [inline content/ssl/edge-certificates/geokey-manager/setup.md]:
```suggestion
You also have access to the `geo_restrictions` parameter, which is mutually exclusive with the `policy` parameter and is part of [Geo Key Manager v1](#geo-key-manager-v1).
```

**Pull request merged**